indigo_app/templates/place/detail.html

```diff
@@ -148,8 +161,24 @@ <h5 class="mb-0">Current tasks ({{ total_open_tasks }})</h5>
           <h5 class="mb-0">Recent activity</h5>
```

Comment:
Can we pull in how far back we're going here, eg. `Recent activity (30 days)` or whatever.